utils/email.py

```diff
@@ -37,3 +38,28 @@ def send_email_with_template(
         send_email_async.send(**kwargs)
     else:
         send_email_async(**kwargs)
+
+
+@dramatiq.actor
```

Comment:
Dramatiq actors should be placed in modules with the specific names: 
https://github.com/Metaculus/metaculus/blob/4554fc104d74ae5bd225224458f9fce6a2623076/metaculus_web/settings.py#L325C1-L325C30

https://github.com/Bogdanp/django_dramatiq?tab=readme-ov-file#declaring-tasks

So this could break async emails, unfortunately